packages/block-editor/src/components/block-list/block-selection-button.js

```diff
@@ -9,12 +9,24 @@ import classnames from 'classnames';
 import { Button } from '@wordpress/components';
 import { useSelect, useDispatch } from '@wordpress/data';
 import { useEffect, useRef } from '@wordpress/element';
-import { BACKSPACE, DELETE } from '@wordpress/keycodes';
+import {
```

Comment:
Worth noting that everything added to this file is directly copied from `WritingFlow`. It is not new code.